charts/cluster-secret/templates/deployment.yaml

```diff
@@ -25,7 +25,10 @@ spec:
     spec:
       securityContext:
         runAsUser: 100 # 100 is set by the container and can NOT be changed here - this would result in a getpwuid() error
-      imagePullSecrets: {{ .Values.clustersecret.clustersecret.imagePullSecrets }}
+      {{- with .Values.clustersecret.clustersecret.imagePullSecrets }}
```

Comment:
this pr will be merged after https://github.com/zakkg3/ClusterSecret/pull/120 
that's why I've changed the base to that branch. 
I am fixing this weird path for values.clustersrcret.clustersecret...

so please change this accordingly (values.imagePull...)